File: Peer/Domain/DataInformation.cs
namespace Peer.Domain;

public class DataInformation
{
    public DataInformation(long countQuery, long querySizeOfBytes)
    {
        CountQuery = countQuery;
        QuerySizeOfBytes = querySizeOfBytes;
    }

    public int Id { get; set; } = 1;
    public long CountQuery { get; set; }
    public long QuerySizeOfBytes { get; set; }
}

File: peerphp/domain/datainformation.php
<?php
class DataInformation {
    public function __construct($countQuery, $querySizeMB) {
        $this->countQuery = $countQuery;
        $this->querySizeMB = $querySizeMB;
    }
    
    public int $id = 1;
    public int $countQuery;
    public float $querySizeMB;
}
?>